dev/com.ibm.ws.jca.cm/resources/com/ibm/ws/j2c/resources/J2CAMessages.nlsprops

```diff
@@ -321,6 +321,11 @@ POOL_MANAGER_NOT_FOUND_J2CA0695=J2CA0695E: Unable to find primary pool manager d
 POOL_MANAGER_NOT_FOUND_J2CA0695.explanation=An internal error occurred during failover processing.
 POOL_MANAGER_NOT_FOUND_J2CA0695.useraction=CMSG0002
 
+# ---------------------------------
+SPNEGO_XA_RECOVERY_NOT_SUPPORTED_J2CA0695W=J2CA0695W: A connection request for XA resource {0} was made using SPNEGO authentication, but no recovery auth data alias was configured. The XA resource will not be able to participate in automated XA recovery unless a recovery auth data alias is configured.
+SPNEGO_XA_RECOVERY_NOT_SUPPORTED_J2CA0695W.explanation=XA recovery is an automated background process, but SPNEGO authentication typically requires manual user input. As a result, SPNEGO authentication can not be used for automatic XA recovery.
+SPNEGO_XA_RECOVERY_NOT_SUPPORTED_J2CA0695W.useraction=Configure a recovery auth data alias on the XA resource.
```

Comment:
Suggested update

>Configure a recovery authentication data alias on the XA resource.

Reply:
Thanks for the comments @Charlotte-Holt. I'll update your other 2 suggestions, but I'm going to decline this one because I think "auth data" is more clear to the user, because in config what the user types in is actually `<authData>` or `xxxAuthDataRef`

Reply:
Got it. That makes sense. Thanks, @aguibert.